src/Files.App/Helpers/Win32/Win32PInvoke.Methods.cs

```diff
@@ -210,5 +210,15 @@ IntPtr lpOverlapped
 			out uint lpBytesReturned,
 			IntPtr lpOverlapped
 		);
+
+		[DllImport("user32.dll")]
+		public static extern int ToUnicode(
```

Comment:
Is this being used?

Reply:
Yeah separated from HotKey class.

Reply:
I'm not sure why should use that though.